espflash/src/error.rs

```diff
@@ -604,13 +607,24 @@ impl<T> ResultExt for Result<T, Error> {
     }
 
     fn for_command(self, command: CommandType) -> Self {
-        match self {
-            Err(Error::Connection(ConnectionError::Timeout(_))) => {
-                Err(Error::Connection(ConnectionError::Timeout(command.into())))
-            }
-            Err(Error::Flashing(ConnectionError::Timeout(_))) => {
-                Err(Error::Flashing(ConnectionError::Timeout(command.into())))
+        fn remap_if_timeout(
+            err: Box<dyn std::error::Error + Send + Sync>,
+            command: CommandType,
+        ) -> Box<dyn std::error::Error + Send + Sync> {
```

Comment:
```suggestion
        fn remap_if_timeout(err: CoreError, command: CommandType) -> CoreError {
```

Can we just use the alias here to clean things up a bit and be consistent, please?